### DISCUSSION TITLE
Accessing different schemas with Accept-Profile

### DISCUSSION THREAD

**Discussion by warflash** [PostgREST/postgrest#2128] [Q&A] [answered]
Hey there!

I'm currently stuck with accessing different schemas of my db, feedback would be massively appreciated.
I added my schema to postgrest.conf like so: `db-schemas = "public, example_schema`, restarted postgres to make sure everything is reloaded and then added `Accept-Profile: example_schema` to my calls.

The response message I'm getting is the following:
``` js
 {
    "hint": null,
    "details": null,
    "code": "42P01",
    "message": "relation \"public.collect\" does not exist"
} 
```
which makes is look like postgrest is not even trying to find and use the schema specified in the request?

Not really sure where to go from here - any hint is appreciated 😊

**Comment by steve-chavez** [comment, marked as answer]
> The response message I'm getting is the following:

Can you share the request you're doing?

**Reply by warflash** [reply]
Sure, here's a screenshot of the setup I'm running: 
It turned out to be a bit all over the place but I tried including everything to make sure I'm not missing something^^
![image](https://user-images.githubusercontent.com/22006082/150404885-2e39bf66-1562-4bf0-b816-bb09722f479e.png)

**Reply by laurenceisla** [reply]
@warflash Does it give you the same response using curl?

```bash
curl "http://localhost:3000/collect" \
  -H "Accept-Profile: bdo_data"
```

**Reply by warflash** [reply]
@laurenceisla Yes, using curl yields the same response:
![image](https://user-images.githubusercontent.com/22006082/150433748-02c3388c-e075-4bd9-be06-510d86fc4023.png)

**Reply by laurenceisla** [reply]
I see. Try changing the schema name to anything else, like: `Accept-Profile: wrong`. If it gives you the same error, then the config file is interpreting it as only one schema, then try completely stopping PostgREST and starting it again.

**Reply by warflash** [reply]
Oh wow.
For some reason the .service file was pointing to a backup version of the postgrest config which did not have the updated schemas 🤦‍♂️ 
Been banging my head against the wall the entire day because of this...
Thank you both for helping out!
